test-jest/cartesian/ReferenceArea.spec.tsx

```diff
@@ -161,16 +160,17 @@ describe('<ReferenceArea />', () => {
         <XAxis dataKey="name" />
         <YAxis tickCount={7} />
         <Bar dataKey="uv" />
-        <ReferenceArea x1="201106" x2="201110" fill="#666" label={{ a: 1 }} />
+        <ReferenceArea x1="201106" x2="201110" fill="#666" label />
       </BarChart>,
     );
 
-    expect(wrapper.find('.recharts-label').length).to.equal(0);
+    expect(container.querySelectorAll('.recharts-label')).toHaveLength(0);
   });
 
-  it('Render custom lable when label is set to react element', () => {
-    const Label = ({ text, ...props }) => <text {...props}>{text}</text>;
-    const wrapper = render(
+  test('Render custom lable when label is set to react element', () => {
+    const Label = ({ text, ...props }: { text: any }) => <text {...props}>{text}</text>;
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { container } = render(
```

Comment:
```suggestion
    render(
```

Please do not disable eslint, instead try to fix it.

Reply:
In this case you use the screen, and the container is unused.